pkgs/os-specific/linux/evdi/default.nix

```diff
@@ -2,15 +2,23 @@
 
 stdenv.mkDerivation rec {
   pname = "evdi";
-  version = "unstable-20210401";
+  version = "unstable-2021-06-11";
 
   src = fetchFromGitHub {
     owner = "DisplayLink";
     repo = pname;
-    rev = "b0b3d131b26df62664ca33775679eea7b70c47b1";
-    sha256 = "09apbvdc78bbqzja9z3b1wrwmqkv3k7cn3lll5gsskxjnqbhxk9y";
+    rev = "65e12fca334f2f42396f4e8d16592d53cab34dd6";
+    sha256 = "sha256-81IfdYKadKT7vRdkmxzfGo4KHa4UJ8uJ0K6djQCr22U=";
   };
 
+  patches = [
+    # Linux 5.13 support
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/DisplayLink/evdi/pull/284.patch";
+      sha256 = "sha256-IjgYhwTG7UYrSN2ZqfTLs27G5n8ZZY5LEH27fF/bF/E=";
```

Comment:
this isn't a stable url they are able to push changes. please use a stable url. E.g.
```
https://github.com/DisplayLink/evdi/pull/284/commits/c5f5441d0a115d2cfc8125b8bafaa05b2edc7938.patch
```